ros_gz_shims/ros_ign_gazebo/launch/ign_gazebo.launch.py.in

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Launch Gazebo Sim with command line arguments."""
+
+from os import environ
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import ExecuteProcess
+from launch.substitutions import LaunchConfiguration
+from launch.conditions import LaunchConfigurationEquals, LaunchConfigurationNotEquals
+
+
+def generate_launch_description():
+    env = {'GZ_SIM_SYSTEM_PLUGIN_PATH':
+           ':'.join([environ.get('GZ_SIM_SYSTEM_PLUGIN_PATH', default=''),
+                     environ.get('LD_LIBRARY_PATH', default='')]),
+           'IGN_GAZEBO_SYSTEM_PLUGIN_PATH':  # TODO(CH3): To support pre-garden. Deprecated.
+                      ':'.join([environ.get('IGN_GAZEBO_SYSTEM_PLUGIN_PATH', default=''),
+                                environ.get('LD_LIBRARY_PATH', default='')])}
+
+    return LaunchDescription([
+        DeclareLaunchArgument('gz_args', default_value='',
+                              description='Arguments to be passed to Gazebo Sim'),
+        # Gazebo Sim's major version
+        DeclareLaunchArgument('gz_version', default_value='@GZ_SIM_VER@',
+                              description='Gazebo Sim\'s major version'),
+
+        # TODO(CH3): Deprecated. Remove on tock.
+        DeclareLaunchArgument(
+            'ign_args', default_value='',
+            description='Deprecated: Arguments to be passed to Gazebo Sim'
+        ),
+        # Gazebo Sim's major version
+        DeclareLaunchArgument(
+            'ign_version', default_value='',
```

Comment:
Shouldn't we also default this one to `GZ_SIM_VER`?

Reply:
The logic I was planning to use was:
- If ign_version is not defined (using the default value), use gz_version (GZ_SIM_VER)
- Otherwise, it means that the user is using a custom ign_version, so we override the behavior.

I'm checking to see if it's not defined. I think it's clearer that way because if we default `ign_version` to GZ_SIM_VER, then it looks like we'd be using the variable, but we're not (unless it's populated)